sklearn/feature_selection/tests/test_base.py

```diff
@@ -1,12 +1,15 @@
 import numpy as np
 from scipy import sparse as sp
+from scipy.stats import pearsonr, spearmanr
```

Comment:
I am unable to use the `np.var` function here because `skb.fit()` takes two parameters `X` and `y` and passes both of them to the score function, but `np.var()` takes only `X`. This gives a problem when I set `axis=0`,
**TypeError: var() got multiple values for keyword argument 'axis'**
Also, I am unable to use `mutual_info_classif` since the input to this has to be modified.